README.md

```diff
@@ -205,3 +205,5 @@ _This should give a result looking something like:_
 #### 9.5 Ready to develop
 
 Use your favorite editor to start working with the code. Read the [contributing guidelines](https://github.com/mxcube/mxcubeweb/blob/develop/CONTRIBUTING.md) and check that linting tools are setup correctly. There are settings [files](https://github.com/mxcube/mxcubeweb/tree/develop/.vscode) for visual studio code in the repository.
+
+Check [code camp exercises]((https://github.com/mxcube/mxcubeweb/wiki/Code-Camp-exercises)https://github.com/mxcube/mxcubeweb/wiki/Code-Camp-exercises) for code examples for starting with mxcube development.
```

Comment:
What do you think about calling it tutorials it seems more common place to me, instead of exercises ? You could add tutorials developed (or done) during code camp or something.

Reply:
Link seems fixed.

Reply:
@marcus-oscarsson I am not sure that I have permissions to "approve" PRs. Maybe it was that I needed to "resolve" the conversation in this case. But I am really not sure.